### ISSUE TITLE
Training without Imagenet weights

### ISSUE THREAD

**Issue by Prabhat-IIT** [facebookresearch/maskrcnn-benchmark#489]
## ❓ Questions and Help

I want to train faster r-cnn without any pretrained weights including Imagenet. Is it possible to do so in current implementation?

**Comment by soumith** [comment]
please use https://discuss.pytorch.org for questions

**Comment by Prabhat-IIT** [comment]
@soumith I think  [ https://discuss.pytorch.org](https://discuss.pytorch.org) is for general pytorch related question. I had to ask if the current configurations allow us to train mask r-cnn or faster r-cnn without any pretrained weights?

**Comment by fmassa** [comment]
@Prabhat-IIT here is an example of a network which is trained without ImageNet weights https://github.com/facebookresearch/maskrcnn-benchmark/blob/master/configs/e2e_mask_rcnn_fbnet.yaml
Generally, you might want to use GroupNorm instead of BatchNorm.

**Comment by soumith** [comment]
it was my mistake @Prabhat-IIT . I misread which repository the question came onto.
